### ISSUE TITLE
U-Net | Predicting Lunar-Craters Using Deep-Learning

### ISSUE THREAD

**Issue by Varunshiyam** [Devanik21/ISRO_Mining_Site_FINAL_APP#93]
This project developed a U-Net model for image segmentation, specifically for identifying lunar craters in images. The model was trained on a dataset of lunar images and corresponding masks, with data augmentation techniques used to improve model robustness. The model was compiled with the Adam optimizer and a dice coefficient loss function, and evaluated using metrics such as IOU and dice coefficient. The trained model was then tested on a separate dataset of lunar images, and also applied to images of lunar boulders to assess its generalization capability.

Kindly, Assign me Under :
- Gssoc-ext
- hacktoberfest-accepted
- Level-3

**Comment by Varunshiyam** [comment]
@Devanik21 I have completed my Issue Kindly review it and Assign me with labels Gssoc-ext , Level-3 .

**Comment by Absk024** [comment]
i would like to contribute @Devanik21
